### DISCUSSION TITLE
1145360哈迪斯

### DISCUSSION THREAD

**Discussion by yassuo295** [pjy612/SteamManifestCache#77] [Share]
[1145360哈迪斯.txt](https://github.com/pjy612/SteamManifestCache/files/13695733/1145360.txt)
内测码7CB6CA29F1083235
联系方式2826821003@qq.com

**Comment by pjy612** [comment]
查收

**Comment by yassuo295** [comment]
感谢代佬&nbsp;


　
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月17日(星期天) 晚上7:15
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [pjy612/SteamManifestCache] 1145360哈迪斯 (Discussion #77)





 
查收
